bin/Powheg/production/2017/13TeV/ggHH_EWChL/powheg_cHHH0.input

```diff
@@ -0,0 +1,118 @@
+use-old-ubound  1    ! if 1 use norm of upper bounding function stored in pwgubound.dat, if present; <> 1 regenerate
+!
+ncall1  10000        ! number of calls for initializing the integration grid
+itmx1       3        ! number of iterations for initializing the integration grid
+ncall2  10000        ! number of calls for computing the integral and finding upper bound
+itmx2       3        ! number of iterations for computing the integral and finding upper bound
+foldcsi     1        ! number of folds on csi integration
+foldy       1        ! number of folds on  y  integration
+foldphi     1        ! number of folds on phi integration
+nubound 10000        ! number of bbarra calls to setup norm of upper bounding function
+# icsimax   1        ! <= 100, number of csi subdivision when computing the upper bounds
+# iymax     1        ! <= 100, number of y subdivision when computing the upper bounds
+# xupbound 2d0       ! increase upper bound for radiation generation
+
+
+! OPTIONAL PARAMETERS
+
+! scale settings:
+#fixedscale   1      ! (dafault 0) Use muren = mufac = 2*m_H
+facscfact   1d0      ! (default 1d0) fac scale factor: mufact = muref * facscfact
+renscfact   1d0      ! (default 1d0) ren scale factor: muren  = muref * renscfact
+
+! Born, plots and powheg strategy:
+#bornonly         1  ! (default 0) if 1 do Born only
+testplots         1  ! (default 0, do not) do NLO and PWHG distributions
+fastbtlbound      1  ! fast evaluation of Btilde upper bounds
+storemintupb      1  ! store function calls in binary file for more precise generation
+ubexcess_correct  1  ! corrects for error in upper bound estimate
+#storeinfo_rwgt 0
+softtest          0  ! no check of the soft limit in the real
+colltest          0  ! no check of the collinear limit in the real
+
+! PARAMETERS needed for ggHH
+hdamp           250
+
+! PARAMETERS for parallel runnning
+! - non parallel (for testing purposes only)
+iseed SEED
+#rand1     0         !  skipping  rand2*100000000+rand1 numbers.  
+#rand2     0         !  (see RM48 short writeup in CERNLIB)              
+
+! - parallel
+#manyseeds 1
+                     ! seeds in the same directory.
+                     ! If set to 1, the program asks for an integer j;
+                     ! The file pwgseeds.dat at line j is read, and the
+                     ! integer at line j is used to initialize the random
+                     ! sequence for the generation of the event.
+                     ! The event file is called pwgevents-'j'.lhe
+#parallelstage 4
+#xgriditeration 1 1
+#maxseeds    1000
+
+! PARAMETERS for reweighting
+#rwl_file 'reweight_setup.xml'
+
+
+! PARAMETERS for Pythia8
+#py8tune  xxx
+#py8hepmc 1
+#py8nohad 1
+#py8nompi 1
+
+check_bad_st1 1
+check_bad_st2 1
+max_io_bufsize  2000000
+
+
+
+
+fakevirt 1
+
+
+#pdfreweight 0
+
+#withnegweights 1 1
+rwl_group_events 2000
+lhapdf6maxsets 50
+rwl_file 'pwg-rwl.dat'
+rwl_format_rwgt 1
```

Comment:
please remove.

Reply:
I'll remove this when I make the template card + small script.